design-document.md

```diff
@@ -23,25 +23,29 @@ _List the most important questions you have about your design, or things that yo
 
 _This is where we work backwards from the customer and define what our customers would like to do (and why). You may also include use cases for yourselves (as developers), or for the organization providing the product to customers._
 
-U1. As a user, I want to be able to create a new budget template and assign different catoegories of expenditures.
+U1. As a user, I want to be able to create a new budget template that lists categories with amounts.
 
-U2. As a user, I want to be able to input and adjust my monthly income as needed.
+U2. As a user, I want to add categories to the budget with amounts.
 
-U3. As an user, I want to be able to track my expenses and see how much funds are available.
+U3. As a user, I want to be able to input and adjust my monthly income as needed.
 
-U4. As a user I want to be able to set and/or delete monthly goals for my expenses and savings.
+U4. As an user, I want to be able to track my expenses and see how much funds are available.
 
-U5. As a user I want to be able to set reminders for bills and upcoming expenses.
+U5. As a user I want to be able to set monthly goals for my expenses and savings.
 
-U6. As a user, I want to be able to view my progress towards goals by some type of bar or pie chart.
+*U6. As a user I want to be able to set reminders for bills and upcoming expenses.
```

Comment:
I'd suggest something like "...set a due date for bills and upcoming expenses". That gives you room to actually log the date (easy) but keep the actual sending of a reminder (harder) out of scope.